README.md

```diff
@@ -136,14 +143,17 @@ All operating systems should now be ready to install TolaActivity source files a
 ```bash
 $ git clone https://github.com/mercycorps/toladata.git
 $ cd toladata
-$ virtualenv -p python3 venv
+$ virtualenv -p python3.8 venv
 $ source venv/bin/activate  # you should see '(venv)' appear on the left of your command prompt
 $ pip install -r requirements.txt
```

Comment:
I think this should be `pip install -r requirements.txt -r dev-requirements.txt` now - we've split out dev and non-dev requirements to simplify server building, but as a dev you'll still want the dev-requirements.  Also @sanjuroj do we want to involve pip-tools at this stage?  (That would make the line `pip install -r requirements.txt` into `pip install pip-tools` then `pip-sync requirements.txt dev-requirements.txt`)?

Reply:
I added a section to install pip-tools and run the pip-sync command. Let me know how that looks and if I should add anything else.